packages/rspack/tests/ConfigTestCases.template.js

```diff
@@ -49,7 +49,7 @@ const createLogger = appendTarget => {
 	};
 };
 
-const TIMEOUT = 10_000;
+const TIMEOUT = 30_000;
```

Comment:
This file is not supposed to be touched. Would you please tell me more about this one?

Reply:
For some reason builds with wasm plugins are taking too long in runners. Previously with 10s it failed both times, but with 30s it seems to pass. Maybe there is another workaround to make build pass in time?

Reply:
This is the error:
```
● ConfigTestCases › builtin-swc-loader › swc-plugin › swc-plugin should compile
  thrown: "Exceeded timeout of 10000 ms for a test while waiting for `done()` to be called.
```

Reply:
I'll also try to find a plugin with target `wasm32-unknown-unknown` and test if it loads faster

Reply:
I've replaced `@swc/plugin-transform-imports` with the simple `@swc/plugin-remove-console` and undid the timeout change. Please rerun the tests to see if it will work

Reply:
Looks good, let's wait for the CI result

Reply:
Should I also repush to undo linter changes on? 😅 They were added automatically with commit hooks